cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/util/ClouderaManagerCommandApiErrorParserUtil.java

```diff
@@ -55,7 +55,7 @@ private ClouderaManagerCommandApiErrorParserUtil() {
                     .forEach(cmd -> errors.addAll(getErrors(cmd, commandsResourceApi)));
 
             if (errors.isEmpty() && StringUtils.isNotEmpty(apiCommand.getResultMessage())) {
-                String reason = String.format("Command [%s], with id [%.0f] failed: %s",
+                String reason = String.format("Command [%s], with id [%s] failed: %s",
```

Comment:
Is `apiCommand.getId()` really a `String`?

Reply:
does it matter?